sdk/search/Azure.Search.Documents/src/Indexes/Models/SearchServiceCounters.cs

```diff
@@ -18,6 +18,7 @@ public partial class SearchServiceCounters
         /// <param name="synonymMapCounter"> Total number of synonym maps. </param>
         /// <param name="skillsetCounter"> Total number of skillsets. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="documentCounter"/>, <paramref name="indexCounter"/>, <paramref name="indexerCounter"/>, <paramref name="dataSourceCounter"/>, <paramref name="storageSizeCounter"/>, or <paramref name="synonymMapCounter"/> is null. </exception>
+        /// <remarks>TODO: Remove this customization once https://github.com/Azure/autorest.csharp/issues/1193 is resolved. </remarks>
```

Comment:
Why is this in the Remarks?  Should it just be a regular comment so it doesn't get pulled into documentation if someone flips the visibility of the ctor?

Reply:
I'll remove it.